tests/mask.py

```diff
@@ -37,7 +37,7 @@
 
 import sys
 import unittest
-import numpy
+import numpy as np
```

Comment:
There was once a discussion about consistently importing numpy as np (and matplotlib.pyplot at plt).  Should we restart it on c.l.o?